### PR TITLE
OCM-4376 | feat: Added the `rosa delete kubeletconfig` sub-command

### PR DESCRIPTION
**Pull request by robpblake**
This PR adds support for the `rosa delete kubeletconfig` sub-command.

### CODE REVIEW

File: cmd/dlt/kubeletconfig/cmd.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	r.Reporter.Debugf("Deleting KubeletConfig for cluster '%s''", clusterKey)
+
+	prompt := fmt.Sprintf("Deleting the custom KubeletConfig for cluster '%s' will cause all non-Control Plane "+
+		"nodes to reboot. This may cause outages to your applications. Do you wish to continue?", cluster.ID())
+
+	if confirm.ConfirmRaw(prompt) {
+
+		err := r.OCMClient.DeleteKubeletConfig(cluster.ID())
+		if err != nil {
+			r.Reporter.Errorf("Failed to delete custom KubeletConfig for cluster '%s': %s",
```
Comment (den-rgb) [inline cmd/dlt/kubeletconfig/cmd.go:61]: `%s`

File: cmd/dlt/kubeletconfig/cmd.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "kubeletconfig",
+	Aliases: []string{"kubelet-config"},
+	Short:   "Delete the custom kubeletconfig for a cluster",
+	Long:    "Delete the custom kubeletconfig for a cluster",
+	Example: `  # Delete the custom kubeletconfig for cluster 'foo'
```
Comment (den-rgb) [inline cmd/dlt/kubeletconfig/cmd.go:35]: is this space necessary

File: cmd/dlt/kubeletconfig/cmd.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+func run(_ *cobra.Command, _ []string) {
+	r := rosa.NewRuntime().WithOCM()
+	defer r.Cleanup()
+
+	clusterKey := r.GetClusterKey()
+	cluster := r.FetchCluster()
+
+	r.Reporter.Debugf("Deleting KubeletConfig for cluster '%s''", clusterKey)
```
Comment (den-rgb) [inline cmd/dlt/kubeletconfig/cmd.go:52]: extra ` at the end

File: cmd/dlt/kubeletconfig/cmd.go (hunk 2 above)
Comment (robpblake) [reply cmd/dlt/kubeletconfig/cmd.go:35]: @den-rgb It's to do with the formatting of the output when you use the `rosa help` command e.g `rosa help delete kubeletconfig`. All other commands have the spaces, so I was just being consistent.

File: cmd/dlt/kubeletconfig/cmd.go (hunk 3 above)
Comment (robpblake) [reply cmd/dlt/kubeletconfig/cmd.go:52]: Thanks. Fixed.

File: cmd/dlt/kubeletconfig/cmd.go (hunk 1 above)
Comment (robpblake) [reply cmd/dlt/kubeletconfig/cmd.go:61]: Thanks. Fixed.

**Comment by den-rgb** [comment]
lgtm

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**